maskit/examples/mnist.py

```diff
@@ -0,0 +1,105 @@
+import tensorflow as tf
+import collections
+import math
+from sklearn.decomposition import PCA
+from pennylane import numpy as np
+from sklearn.preprocessing import minmax_scale
+from maskit.examples.utils import Data
+
+MAX_TRAIN_SAMPLES = 11471
+MAX_TEST_SAMPLES = 1952
+
+
+def reduce_image(x: np.ndarray) -> np.ndarray:
```

Comment:
Some of the functions provided in this implementation might be worth adding to `utils`. But this is just a comment :) we can still leave this for our future selfs when we realise that we can reuse some of those things.